.github/workflows/merge_conflicts.yml

```
@@ -13,4 +13,5 @@ jobs:
         uses: eps1lon/actions-label-merge-conflict@releases/2.x
         with:
           dirtyLabel: "merge-conflict"
+          commentOnDirty: "This pull request has conflicts, please resolve them."
```

This might be a bit more instructive for newer contributors.
```suggestion
          commentOnDirty: "This pull request has conflicts, please merge `master` and resolve them."
```

---

Thanks!